art_show/templates/art_pieces_form.html

```diff
@@ -0,0 +1,368 @@
+                  <input type="checkbox" name="no_quick_sale" value="1"{% if real_piece.no_quick_sale %} checked{% endif %} />
+                  I don't want my piece to be for sale after bidding ends.</label>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="modal-footer">
+          <button type="submit" class="btn btn-primary save_piece">Save</button>
+          <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+        </div>
+      </form>
+    </div>
+  </div>
+</div>
+{%- endmacro -%}
+
+<iframe id="upload_frame" name="upload_frame" style="display:none"></iframe>
+
+<div id="piece_modals">
+  {{ piece_modal() }}
+  {% if app.art_show_pieces -%}
+  {% for piece in app.art_show_pieces -%}
+  {{ piece_modal(piece) }}
+  {%- endfor %}
+  {%- endif %}
+</div>
+
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">Art Show Information</h3>
+  </div>
+  <div class="panel-body">
+    <form method="post" action="../art_show_applications/edit" role="form" class="form-horizontal">
+      <input type="hidden" name="id" value="{{ app.id }}" />
+      <input type="hidden" name="return_to" value="{{ c.PAGE_PATH }}" />
+      {{ csrf_token() }}
+      {{ macros.form_group(app, 'banner_name', label="Banner Name", help="The name displayed with your art.", placeholder=app.artist_name or app.attendee.full_name) }}
+      {{ macros.form_group(app, 'check_payable', label="Name on Check", help="The name your check should be made payable to.", placeholder=app.attendee.legal_name or app.attendee.full_name) }}
+      {{ macros.form_group(app, 'hotel_name', label="Hotel Staying At") }}
+      {{ macros.form_group(app, 'hotel_room_num', label="Room Number") }}
+
+      <div class="form-group">
+        <div class="col-sm-6 col-sm-offset-3">
+          <div class="input-group">
+            <button type="submit" class="btn btn-primary">Update Info</button>
+          </div>
+        </div>
+      </div>
+    </form>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">Art Show Pieces</h3>
+  </div>
+  <div class="panel-body">
+    <div class="row text-center">
+      <div class="col-sm-4">
+        <button class="btn btn-primary" id="add_piece_modal">Add {% if app.art_show_pieces %}Another {% endif %}Piece</button>
+      </div>
+      <div class="col-sm-4">
+        <form method="post" action="../art_show_applications/confirm_pieces" class="form-inline">
+          <input type="hidden" name="id" value="{{ app.id }}" />
+          <input type="hidden" name="return_to" value="{{ c.PAGE_PATH }}" />
+          {{ csrf_token() }}
+          <button type="submit" class="btn btn-default" id="confirm_pieces">Confirm Current Pieces</button>
+        </form>
+      </div>
+      <div class="col-sm-4" data-toggle="tooltip" title="Coming Soon!">
+        <button class="btn btn-success" disabled>Print Check-In Form</button>
```

Comment:
This button doesn't do anything because we don't have specs for the check-in form, but having it present but disabled is a good way to let artists know to come back to the page.